Python/Endpoint Examples/JSON Payload/README.md

```diff
@@ -0,0 +1,10 @@
+This directory contains examples of how to call each endpoint of the
+PdfRest API using Python. Please refer to https://pdfrest.com/documentation.html
+for accepted inputs, options, and other specifics about individual endpoints.
+
+#### How to run Python samples
+
+Using Python3:
+
+1. `cd 'Python/Endpoint Examples/JSON Payload/`
```

Comment:
This is missing the closing apostrophe, same thing in the Multipart Payload readme
```suggestion
1. `cd 'Python/Endpoint Examples/JSON Payload/'`
```